src/assets/messages.json

```diff
@@ -3363,7 +3366,8 @@
     "blocks-config-accuracy_threshold_meters-type": "Le seuil de précision en mètres devrait être un nombre",
     "blocks-config-accuracy_timeout_seconds-type": "Le délai d'attente de précision en quelques secondes devrait être un nombre",
     "blocks-config-value-type": "L'expression de sortie devrait être une chaîne",
-    "blocks-config-is_member-type": "La valeur / l'expression devrait être une chaîne"
+    "blocks-config-is_member-type": "La valeur / l'expression devrait être une chaîne",
+    "blocks-values-0-value-pattern": "Le contenu ne doit contenir que le script, les chiffres et la ponctuation de base"
```

Comment:
I know this was from the `getErrorMessageLocalizationKeyForProperty` logic. Is there a way to update that method so we will have `resources` word in this localization key ? Kind of `blocks-resources-values-x-value-pattern`